parlai/agents/hugging_face/t5.py

```diff
@@ -238,10 +247,7 @@ def forward(
             self.stack.parallelize()
         mask = input != self.padding_idx
         outputs = self.stack(input, attention_mask=mask, output_hidden_states=False)
-        for k in outputs:
-            if torch.is_tensor(outputs[k]):
-                outputs[k] = outputs[k].to(input.device)
-        return outputs[0], mask
+        return outputs[0].to(input.device), mask
```

Comment:
Cool. Maybe leave a comment saying future uses of outer outputs should also to() the tensors.